server/Routes/auth.js

```diff
@@ -0,0 +1,85 @@
+const express = require('express');
+
+const {
+    Client
+} = require('pg');
+const bcrypt = require('bcrypt');
+const client = new Client({
+    user: "postgres",
+    password: "123",
+    host: "localhost",
+    port: 6000,
+    database: "papel"
```

Comment:
these values should be stored in the environment variable since you must keep them secret